### PR TITLE
InfluxDB Enterprise 1.8.5 release notes

### PR DESCRIPTION
**Pull request by kelseiv**
Please see questions inline below- thank you!

### CODE REVIEW

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md [hunk 1]
```diff
@@ -9,11 +9,24 @@ menu:
...
+## v1.8.5 [2020-04-19]
+
+The InfluxDB Enterprise 1.8.5 release builds on the InfluxDB OSS 1.8.5 release.
+For details on changes incorporated from the InfluxDB OSS release, see
+[InfluxDB OSS release notes](/influxdb/v1.8/about_the_project/releasenotes-changelog/#v185-2021-04-19).
+
+### Bug fixes
+
+- Resolve TSM backup "snapshot in progress" error.
+- Make meta queries respect QueryTimeout values.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:21]: Is line 21 same as this (mentioned in OSS 1.8.5 release notes):

* Update meta queries (for example, SHOW TAG VALUES, SHOW TAG KEYS, SHOW SERIES CARDINALITY, SHOW MEASUREMENT CARDINALITY, and SHOW MEASUREMENTS) to check the query context when possible to respect timeout values set in the query-timeout configuration parameter. Note, meta queries will check the context less frequently than regular queries, which use iterators, because meta queries return data in batches.

Comment (kelseiv) [reply content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:21]: If yes, I believe we can drop here?

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md [hunk 2]
```diff
@@ -9,11 +9,24 @@ menu:
...
+- Add labels to verbose output.
+- Always forward repairs to next node.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:23]: meta and/or data node?

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md (hunk 1 above)
Comment (davidby-influx) [reply content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:21]: It is the same, just making the new feature work in Enterprise, so feel free to drop here (or drop in OSS, whichever you prefer).

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md [hunk 3]
```diff
@@ -9,11 +9,24 @@ menu:
...
+
+The InfluxDB Enterprise 1.8.5 release builds on the InfluxDB OSS 1.8.5 release.
+For details on changes incorporated from the InfluxDB OSS release, see
+[InfluxDB OSS release notes](/influxdb/v1.8/about_the_project/releasenotes-changelog/#v185-2021-04-19).
+
+### Bug fixes
+
+- Resolve TSM backup "snapshot in progress" error.
+- Make meta queries respect QueryTimeout values.
+- Add labels to verbose output.
```
[the remark this answers is not in the document]
Comment (davidby-influx) [reply content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:22]: From Ayan, of lamented loss to Influx:

> Prior to this commit, the show shards command will print the shard
> owners value without labels like below:
> 
> {7 ics106019162.ics-eu-1.asml.com:8088} {4 ics106019220.ics-eu-1.asml.com:8088}]
> 
> This is because we use the "%v" verb to print the struct.
> 
> This patch uses the "%+v" verb which instructs Go to include the struct
> member names.
> 
> This will allow for more readable output like below:
> 
> [{ID:7 TCPAddr:ics106019162.ics-eu-1.asml.com:8088} {ID:4 TCPAddr:ics106019220.ics-eu-1.asml.com:8088}]

TLDR from me: 

> This change will affix labels to the values returned in `SHOW SHARDS` output clarifying what is the node ID and what is the TCP address.

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md (hunk 2 above)
Comment (davidby-influx) [reply content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:23]: As I understand what Mr. Norton meant, I would write: "...to the next data node, even if the current data node does not have to take any action itself for this repair"

**Review by davidby-influx — APPROVED** [review]
Hopefully, I have clarified things.

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md (hunk 1 above)
Comment (kelseiv) [reply content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:21]: Thank you, @davidby-influx, if added to both OSS 1.8.5 and Enterprise 1.8.5, will drop here.

**Comment by kelseiv** [comment]
> Hopefully, I have clarified things.

Yes!
